### ISSUE TITLE
Jump-to-definition shows call sites when function is invoked with argument

### ISSUE THREAD

**Issue by wearhere** [babel/babel-sublime#337]
This package appears to incorrectly count function calls as definitions when those function calls involve arguments. In this screenshot, I have invoked jump-to-definition on the second call to `foo`, on the last line. You can see that the menu shows both the function's definition and the function's first call, but not the second call:

<img width="400" alt="screen shot 2017-07-21 at 7 16 37 pm" src="https://user-images.githubusercontent.com/110000/28487621-64f4b83e-6e49-11e7-886a-db7cffd72aee.png">

If I add an argument to `foo`'s second call, then it will appear in the list too:

<img width="314" alt="screen shot 2017-07-21 at 7 19 22 pm" src="https://user-images.githubusercontent.com/110000/28487626-83d25eaa-6e49-11e7-9708-2530f335d36c.png">

If I switch the file from being parsed using Babel to the regular JavaScript package this issue goes away (invoking jump-to-definition will jump to the actual definition, no list will appear).

**Comment by wearhere** [comment]
I would be very interested in fixing this issue since it pretty much breaks jump-to-definition and the Cmd-R method list for me, if the maintainer could provide some leads.

**Comment by wearhere** [comment]
Ok I've found that whatever the issue here is, is fixed on `master`: I uninstalled Babel from Package Control, then symlinked a checkout of this repo into my Packages directory, and the issue went away. So can you simply publish a new version of this package?

**Comment by neaumusic** [comment]
I can't figure out how to manually install babel, but I'm desperate to fix the goto_definition problem

**Comment by Thom1729** [comment]
It looks like the last release was over two years ago, which probably bodes poorly for future releases.

I'm working on a new extended JavaScript package: [JS Custom](https://github.com/Thom1729/Sublime-JS-Custom). It's not in Package Control yet, but it just reached beta. It includes JSX support and does not suffer from the above bug. It's intended as a modern replacement for Babel Sublime and similar packages.

I'd appreciate any feedback, including feature requests, and I'm available for questions on the [Sublime Discord channel](https://discord.gg/D43Pecu).

**Comment by borela** [comment]
Try FJSX15 https://github.com/borela/naomi Jump to definition is working + many stage-0 features.

**Comment by wearhere** [comment]
To expand on my [fix](https://github.com/babel/babel-sublime/issues/337#issuecomment-317148670) above, since this is still happening :\

```sh
# First uninstall the Babel package. Then:

cd ~/Application\ Support/ Sublime\ Text\ 3/Packages
git clone git@github.com:babel/babel-sublime.git Babel
```

(More information on overriding packages [here](https://packagecontrol.io/docs/customizing_packages)).

**Comment by Thom1729** [comment]
This should be fixed in v10.0.0-beta.1. See #379 for more information on the new release. Please report any problems with v10 in a new issue.
